wip with ClayModal

ambrinchaudhary opened the pull request:
Check https://issues.liferay.com/browse/LPS-88193

brandizzi committed:
LPS-88344 portal-search-engine-adapter-api: Add a rescore query to search request so we can use the Rescorer API.

https://issues.liferay.com/browse/LPS-88344

Co-authored-by: Adam Victor Nazareth Brandizzi <adam.brandizzi@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

brandizzi committed:
LPS-88344 portal-search-elasticsearch6-impl: Get query we added to search context and use it as a rescorer query.

https://issues.liferay.com/browse/LPS-88344

Co-authored-by: Adam Victor Nazareth Brandizzi <adam.brandizzi@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

ambrinchaudhary committed:
LPS-87788 Open Clay Modal

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

4lejandrito committed:
LPS-88179 Setting company scope

4lejandrito committed:
LPS-88179 Adding companyId to every OAuth2Manager method

ambrinchaudhary committed:
LPS-87788 Loading

ambrinchaudhary committed:
LPS-87788 Call backend service to get the common tags

adolfopa committed:
LPS-88396 Create skeleton resources and models

adolfopa committed:
LPS-88396 Support Cookie authorization

fortunatomaldonado committed:
LPS-88170 Rename column and finder names to prevent syntax errors in generated classes

adolfopa committed:
LPS-88396 Remove old actions

adolfopa committed:
LPS-88396 Move logic into resources

adolfopa committed:
LPS-88396 Create context providers

fortunatomaldonado committed:
LPS-88170 Only rename template variable

ajsampang67 committed:
LPS-88353 changed form field from input to textarea

ambrinchaudhary committed:
LPS-87788 Changes in soy template

adolfopa committed:
LPS-88396 Create skeleton module for Bulk REST API

Commit:
LPS-87743 Adding change tracking client API skeleton

nikki-pru committed:
LRQA-45152 Fix path

nikki-pru committed:
LRQA-45152 Use Form Fields macro

Commit:
LPS-88083 users-admin-web - Update organization site design

Commit:
LPS-88083 users-admin-web - Build lang

Commit:
LPS-88448 Populate custom field options

ambrinchaudhary committed:
LPS-88396 Adapt backend services

ambrinchaudhary committed:
LPS-87788 New method

hudakl committed:
LPS-88434 Use the original categoryIds if there's no permissionChecker available

jcampoy committed:
LPS-88477 Encapsulate logic for getting the article from portletPreferences

jcampoy committed:
LPS-88477 Take in account preferences for embedded portlets

hhuijser committed:
LPS-86806 Refactor the way we handle line breaks in new class instantiation

ealonso committed:
LPS-86125 Reorder

ealonso committed:
LPS-88580 Adds missing submodule

ealonso committed:
LPS-86125 Reorder by path, feel free to remove this commit if you prefer order by module name

ealonso committed:
LPS-88440 Update Fjord with new path for images

ealonso committed:
LPS-88440 Uses fragment repository to store images

ealonso committed:
LPS-88440 Also use repository to save the thumbnail images

ealonso committed:
LPS-88440 Adds new module to update resources

ealonso committed:
LPS-88440 Register the processor

ealonso committed:
LPS-88440 Implements processor, replace images that contains "../resources/" for the corresponding image in the repository

ambrinchaudhary committed:
LPS-87788 Use ClayMultiSelect component

ealonso committed:
LPS-88440 We need to add permission because of DL checking when we are using images into a repository

ealonso committed:
LPS-88440 Auto

ealonso committed:
LPS-88440 Auto

pavel-savinov committed:
LPS-88433 Show approved badge when article has an approved version but not in an approved state right now

ealonso committed:
LPS-88440 Export images to resource folder

ealonso committed:
LPS-88440 Update Westeros with new path for images

ealonso committed:
LPS-88440 Uses fragment repository to store images

ealonso committed:
LPS-88440 Also use repository to save the thumbnail images

pavel-savinov committed:
LPS-88433 Rearrange journal article actions menu following new UX proposal

pavel-savinov committed:
LPS-88433 Missing language keys

pavel-savinov committed:
LPS-88433 Buildlang

ealonso committed:
LPS-88440 Delete images when we delete the collection

ealonso committed:
LPS-88440 Import images from zip

ealonso committed:
LPS-88440 Delete the preview image if we have when we are deleting the fragment entry

ealonso committed:
LPS-88440 Replace image if we have one with the same name

diegonvs committed:
LPS-86128 Expose message-boards-web module name using NPMResolver

diegonvs committed:
LPS-86128 Replace metal-cli on message-boards module

diegonvs committed:
LPS-86128 SF

diegonvs committed:
LPS-86128 Update metal dependencies

diegonvs committed:
LPS-86128 Update lockfiles

wanderlast committed:
LPS-88526 - log level should be error, not warn. Reverts LPS-74437

lesliewong92 committed:
LRQA-45611 Remove logic to create PMDBatchTestClassGroup's in factory method

lesliewong92 committed:
LRQA-45611 Remove class PMDBatchTestClassGroup

jonathanmccann committed:
LPS-88550 There may be more than two entries with the same fileVersionId due to LPS-30576

hhuijser committed:
LPS-86806 Fix logic

hhuijser committed:
LPS-86806 Add parsing logic for interfaces

kyle-miho committed:
LRQA-43776 Add asset lists as a testray component

kyle-miho committed:
LRQA-43776 Add new macros for asset lists

arboliveira committed:
LPS-88344 portal-search: Add a rescore query to search request so we can use the Rescorer API.

https://issues.liferay.com/browse/LPS-88344

Co-authored-by: Adam Victor Nazareth Brandizzi <adam.brandizzi@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

arboliveira committed:
LPS-88344 portal-search-api: Add a rescore query to search request so we can use the Rescorer API.

https://issues.liferay.com/browse/LPS-88344

Co-authored-by: Adam Victor Nazareth Brandizzi <adam.brandizzi@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

kyle-miho committed:
LRQA-43776 Add needed paths

kyle-miho committed:
LRQA-43776 Add asset lists test cases

kyle-miho committed:
LRQA-43776 Add tasks and descriptions to testcases

dantewang committed:
LPS-88557 Explicitly append Whip jar path to test.classpath

hhuijser committed:
LPS-86806 Also handle enum and annotation definition

hhuijser committed:
LPS-86806 Add parsing logic for annotation fields

hhuijser committed:
LPS-86806 Add class JavaInstanceInitialization

hhuijser committed:
LPS-86806 Add class JavaStaticInitialization

Commit:
LPS-87985 - Updating Alloy 3.1.x to c90858c85276350cb30a2ec462bb40a358171c0f

liulukiwi committed:
LRQA-45371 Replace the lar with updated private URL in wiki page link

cgoncas committed:
LPS-88562 Unused dependencies

cgoncas committed:
LPS-88562 Add new content-space-apio-client-test-util module

cgoncas committed:
LPS-88562 New ContentSpaceApioTestUtil class.

jbalsas committed:
LPS-86128 Follows RESOLVED_MODULE_NAME existing pattern

cgoncas committed:
LPS-88562 Use it.

jbalsas committed:
LPS-86128 Injects the module resolved name for admin portlet too

jbalsas committed:
LPS-86128 SF

ealonso committed:
LPS-88567 Adds new method to know if we have resources or not

ealonso committed:
LPS-88567 Avoid unnecessary calls

ealonso committed:
LPS-88567 Extract into a method and reuse it

ealonso committed:
LPS-88567 Adds resources folder if it doesn't exist

ealonso committed:
LPS-88567 Reuse code

adolfopa committed:
LPS-86128 SF - Remove cast

adolfopa committed:
LPS-86128 Auto SF

ambrinchaudhary committed:
LPS-87788 Call to backend services must be done by the new module

jorgediaz-lr committed:
LPS-88570 Add error trace in case of unexpected error in ExportImportLocalServiceImpl.

ambrinchaudhary committed:
LPS-87788 Adapt tags to ClayMultiSelect component

4lejandrito committed:
LPS-88179 Throwing PortalException

4lejandrito committed:
LPS-88179 Adding companyId to DLOpenerGoogleDriveManager

4lejandrito committed:
LPS-88179 Passing companyId to the oauthManager

4lejandrito committed:
LPS-88179 Passing companyId to the dlOpenerGoogleDriveManager

4lejandrito committed:
LPS-88179 Renaming configuration class

adolfopa committed:
LPS-86128 Use render request, as in Wiki and Segments

cgoncas committed:
LPS-88562 Semantic versioning

cgoncas committed:
LPS-88562 Extract common href to setup

cgoncas committed:
LPS-88562 Use JSON files, instead strings.

cgoncas committed:
LPS-88562 Better test.

victorg1991 committed:
LPS-88164 Add test for permissions in blog api

ealonso committed:
LPS-88567 Adds new view for collection resources

ealonso committed:
LPS-88567 Adds action to delete resources

Commit:
LPS-88471 - Update Clay to v2.5.1

Commit:
LPS-88471 - Update clay components new version API

Commit:
LPS-88471 - Update clay components new version usage

ealonso committed:
LPS-88567 buildLang

ealonso committed:
LPS-88567 Allow users navigate to view resources view

4lejandrito committed:
LPS-88179 Adding navigator entry

4lejandrito committed:
LPS-88179 Using service name to retrieve the config

Commit:
LPS-88471 - SF

ealonso committed:
LPS-88567 Adds new friendly url route

diegonvs committed:
LPS-86132 Replace metal-cli on site-navigation-admin-web

diegonvs committed:
LPS-86132 Expose module name for dynamic module name and version on JSPs

diegonvs committed:
LPS-86132 Update lockfiles

ealonso committed:
LPS-88567 Create a var for resource folder id

diegonvs committed:
LPS-86132 Rename WebKey to follow RESOLVED_MODULE_NAME pattern

diegonvs committed:
LPS-86132 Remove cast

4lejandrito committed:
LPS-88179 Claring map on deactivate

4lejandrito committed:
LPS-88179 Synchronizing _getGoogleAuthorizationCodeFlow

Commit:
LPS-88131 users-admin-web - Update announcements design

shuyangzhou committed:
LPS-88170 Less local variables, less chance to conflict

pei-jung committed:
LPS-88083 Replace <aui:field-wrapper> with form-group divs for proper spacing between form elements

yichenroy committed:
LRQA-45457 Add failure message generator for git-lpush failures

petershin committed:
LPS-88588 Loosen regex pattern to check for spaces or a tab

pei-jung committed:
LPS-88083 We only need one form section here

shuyangzhou committed:
LPS-86836 Setup for sub-jvms

michaelhashimoto committed:
LRQA-45126 Add ability to use a specific java home based off of the batch name

michaelhashimoto committed:
LRQA-45126 Add ability to use a specific set of java opts for tomcat bundles based off of the batch name

michaelhashimoto committed:
LRQA-45126 Add a new batch for testing functional smoke tests with open11

mtambara committed:
LPS-87997 Protect against directory traversal

michaelhashimoto committed:
LRQA-45126 SF

brianchiu committed:
LRQA-43776 Use more explicit naming

brianchiu committed:
LRQA-43776 No need if we are only doing this once

brianchiu committed:
LPS-87422 Add test for seeing asset list usage

brianchiu committed:
LRQA-43776 Harmonize wording

brianchiu committed:
LRQA-43776 SF

brianchiu committed:
LRQA-43776 This should be committed under a different ticket

zhangyan0701 committed:
LRQA-45395 Update macro to select frame before input page name

liulukiwi committed:
LRQA-45371 Add pause to wait clicking continue finished

brianchandotcom committed:
LPS-88440 This should be a 1.0.0 since it's not getting backported

brianchandotcom committed:
LPS-88440 SF

brianchandotcom committed:
LPS-88588 prep next

brianchandotcom committed:
LPS-88562 Avoid chaining

brianchandotcom committed:
LPS-86128 SF

brianchandotcom committed:
LPS-86128 SF

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 851126f86320ab6785398eb12417c3b5f47e6756

hhuijser committed:
LPS-86806 Improve logic to retrieve end position

hhuijser committed:
LPS-86806 Add logic for parsing enum constant definitions

Hanlf committed:
LRQA-45405 Select top frame before going to other page

JoyceWang08211 committed:
LRQA-45653 Remove the unnecessary variable declare

hhuijser committed:
LPS-86806 Add logic for do/while statements

liulukiwi committed:
LRQA-45064 Add view Web Content Structures permission in case

jbalsas committed:
LPS-86132 SF

Commit:
LPS-88047 Update packageinfo

liulukiwi committed:
LRQA-45042 Remove variable singleSiteTypePermission in case

ealonso committed:
LPS-88433 Rename to cssClass

ealonso committed:
LPS-86132 Auto SF

ealonso committed:
LPS-86132 SF - Moves code to display context

4lejandrito committed:
LPS-88179 Merging impl and web modules

Commit:
LPS-64468 Add tooltip to custom mapping LDAP fields

Commit:
LPS-88448 Remove unnecessary code

ambrinchaudhary committed:
LPS-87788 Extract the added and removed tags and refactor the fetch method

Commit:
LPS-88448 Retrieve the latest registered version of the services.

When a custom field is added, the entityModel is updated and registered again, and so does the criteriaContributor.

Commit:
LPS-88471 - Baseline

4lejandrito committed:
LPS-88179 No longer needed

jpince committed:
LPS-87850 Brian Auto SF

cgoncas committed:
LPS-88604 Fix flaky tests

pavel-savinov committed:
LPS-88613 Add service builder descriptor for asset entry usage entity

pavel-savinov committed:
LPS-88613 Initial service code

pavel-savinov committed:
LPS-88613 Implement basic methods for asset entry usage entity

Commit:
LPS-88448 Add test

pavel-savinov committed:
LPS-88613 Regenerate service

cgoncas committed:
LPS-88617 We don't need so many test utils, all test util will be included in portal-apio-test-util

pavel-savinov committed:
LPS-88613 Implement asset entry usage comparator

pavel-savinov committed:
LPS-88613 Export service-related packages

pavel-savinov committed:
LPS-88613 Semantic versioning

xbrianlee committed:
LRQA-45536 Ignore additional failing SAML test

thektan committed:
LPS-88003 Add tests

thektan committed:
LPS-88003 Support links on button

thektan committed:
LPS-88003 Support more labels on ClayToggle

thektan committed:
LPS-88003 Update segments edit form layout

thektan committed:
LPS-88003 Add language keys

4lejandrito committed:
LPS-88179 Regen

ealonso committed:
LPS-88613 SF

thektan committed:
LPS-88003 Blur input on enter

shuyangzhou committed:
Revert "LPS-86836 Setup for sub-jvms"

This reverts commit ebc08bd6e6f999cb0a6f49387b317f089f47bcda.

shuyangzhou committed:
Revert "LPS-86836 Remove redundant setting, it is already included in InitUtil.init()"

This reverts commit 86b075324c4400140208ccccdeb7293467f8ac55.

shuyangzhou committed:
Revert "LPS-86836 Escape liferay home path when initializing Log4J"

This reverts commit 67e9ff12ae58bebec39cd86f122bc24a9f63822f.

shuyangzhou committed:
Revert "LPS-86836 Need to register Html service manually after moving the initialization of HtmlUtil from Spring to InitUtil"

This reverts commit f9104be4490905610a94b287423726aa7dcd8445.

shuyangzhou committed:
Revert "LPS-86836 Move the initialization of HtmlUtil from spring to InitUtil"

This reverts commit 84ae57780853bf9107e0c493a4d4ba17c9ab3254.

thektan committed:
LPS-88003 Use current locale

nikki-pru committed:
LRQA-45152 Set tag variable

shuyangzhou committed:
LPS-86836 Escape just for xml attribute. (> is allowed for xml attribute)

Commit:
LPS-87743 Auto SF

yichenroy committed:
LRQA-45457 Add lpush failure message generator to top level build

brianchiu committed:
LRQA-45666 Wait for side bar to finish loading before clicking it

brianchandotcom committed:
LPS-86836 Rename and sort

brianchandotcom committed:
LPS-86836 auto SF

4lejandrito committed:
LPS-88179 SF

ealonso committed:
LPS-88613 Fix exports

ealonso committed:
LPS-88613 Semantic versioning

ealonso committed:
LPS-88613 dependencies

yichenroy committed:
LRQA-45206 Make rebase error failure snippet more precise

patriciajeanperez committed:
LPS-87587 Update macro to allow reading of notification

patriciajeanperez committed:
LPS-87587 Add macro for viewing Shared With Me

patriciajeanperez committed:
LPS-87587 Add test for navigating to Shared with Me via notification

patriciajeanperez committed:
LPS-87587 SF

patriciajeanperez committed:
LRQA-45152 Use existing macro

xbrianlee committed:
LRQA-45536 Ignore test due to LPS-88487

pyoo47 committed:
LRQA-45457 GitLPushFailureMessageGenerator needs to come before CIFailureMessagGenerator

pyoo47 committed:
LRQA-45457 Wordsmith

brianchandotcom committed:
LPS-88382 Temp revert

brianchandotcom committed:
LPS-88382 release temp revert

brianchandotcom committed:
LPS-88382 release temp revert

brianchandotcom committed:
LPS-88003 SF

shuyangzhou committed:
LPS-88382 Always add util-taglib bundle.

shuyangzhou committed:
Revert "LPS-88382 As we provide an implementation for http://java.sun.com/jsp/jstl/core it should not be blacklisted and it must be added as a requirement."

This reverts commit 0682999f5ce2d1aa26349188ad3415eda7d04495.

joshchong committed:
LRQA-45672 Reduce flakiness

shuyangzhou committed:
Revert "LPS-88382 As we provide an implementation for http://java.sun.com/jsp/jstl/core ensure that it's added as a required capability when used."

This reverts commit 9cac2914aad0da7b1dbe0a1d3891ea6354cfb884.

brianchandotcom committed:
LRQA-45457 Sort

shuyangzhou committed:
Revert "LPS-88377 Use AUI taglib to force jsp analyzdr to include JSTL"

This reverts commit c3635741387f3e6fcca0b1277376c3144f7fed7b.

brianchandotcom committed:
LRQA-45457 prep next

brianchandotcom committed:
LPS-87743 Rename, sounds robotic, but it's consistent

brianchandotcom committed:
LPS-87743 Sort

brianchandotcom committed:
LPS-86108 Why is there a web in this module? See ac1863153c8d9263d99c10316b153de7c758a5ca

brianchandotcom committed:
LPS-86125 Fix trailing comma

brianchandotcom committed:
LPS-88382 release temp revert

brianchandotcom committed:
LPS-88382 release temp revert

brianchandotcom committed:
LPS-88448 Hugo SF

brianchandotcom committed:
LPS-88448 Sort

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 22047696fa51054d684cc31b672ee5f9f6907d67

shuyangzhou committed:
LPS-88382 Use latest ant bnd

michaelhashimoto committed:
LRQA-45639 Set httpusecaches to false when finding the patching tool version

brianchandotcom committed:
LPS-88382 prep next

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as a5600811fb6ea46c021eae0225b68a1704af98bf

brianchandotcom committed:
LPS-88170 Better name

brianchandotcom committed:
LPS-88170 prep next

brianchandotcom committed:
LPS-88170 prep next

brianchandotcom committed:
LPS-88170 Regen

brianchandotcom committed:
LPS-88164 Rename to match our usual pattern. BaseXyz where Base* is abstract, and concrete classes are HelloXyz and YoXyz

adolfopa committed:
LPS-86128 Semver

ambrinchaudhary committed:
LPS-87788 Call to backend service that updates the tags

ambrinchaudhary committed:
LPS-87788 Disable autocomplete in tagSelector. Will be implemented as part of another story.
